core/src/blocktree.rs

```diff
@@ -3,121 +3,80 @@
 //! access read to a persistent file-based ledger.
 
 use crate::entry::Entry;
+#[cfg(feature = "kvstore")]
+use crate::kvstore;
 use crate::packet::{Blob, SharedBlob, BLOB_HEADER_SIZE};
 use crate::result::{Error, Result};
+
 use bincode::{deserialize, serialize};
-use byteorder::{BigEndian, ByteOrder, ReadBytesExt};
+
 use hashbrown::HashMap;
-use rocksdb::{
-    ColumnFamily, ColumnFamilyDescriptor, DBRawIterator, IteratorMode, Options, WriteBatch, DB,
-};
-use serde::de::DeserializeOwned;
+
+use rocksdb;
```

Comment:
Should this have a: `#[cfg(not(feature = "kvstore"))]`

Reply:
Yes, fixing